resources/monitor_activity_logs.go

```diff
@@ -244,8 +244,16 @@ func fetchMonitorActivityLogs(ctx context.Context, meta schema.ClientMeta, paren
 	if err != nil {
 		return err
 	}
+	// azure returns same events sometimes so we have to filter out the duplicates
+	seen := make(map[string]struct{})
```

Comment:
why not map[string]bool ?

Reply:
It doesn't matter here, but map[string]struct{} consumes less memory than map[string]bool if all we want to test if an key is present in map or not.